DQMOffline/L1Trigger/src/L1TTauOffline.cc

```diff
@@ -533,10 +533,21 @@ bool L1TTauOffline::matchHlt(edm::Handle<trigger::TriggerEvent> const& triggerEv
 
   trigger::TriggerObjectCollection trigObjs = triggerEvent->getObjects();
 
-  for (auto trigIndexIt = m_trigIndices.begin(); trigIndexIt != m_trigIndices.end(); ++trigIndexIt) {
+  for (vector<int>::const_iterator trigIndexIt = m_trigIndices.begin(); trigIndexIt != m_trigIndices.end();
+       ++trigIndexIt) {
     const vector<string> moduleLabels(m_hltConfig.moduleLabels(*trigIndexIt));
-    const unsigned moduleIndex = m_hltConfig.size((*trigIndexIt)) - 2;
-
+    // V.M. 2023.03.08 Same issue as in the L1TMuonDQMOffline.cc - some modules are behind hltBoolEnd, but we want the last one before the "hltBoolEnd"
+    unsigned int moduleIndex = 999999;
+    for (int ii = 0; ii < int(moduleLabels.size()); ii++) {
+      if (moduleLabels[ii] == "hltBoolEnd") {
+        moduleIndex = ii - 1;
+        break;
+      }
+    }
+    if (moduleIndex == 999999) {
+      edm::LogError("L1TMuonDQMOffline") << "Found no module label in trigger " << (*trigIndexIt) << endl;
```

Comment:
```suggestion
      edm::LogError("L1TMuonDQMOffline") << "Found no module label in trigger " << trigIndex;
```
(If you go for the cleaner range based for loop on the trigger indices)
By the way, there is no need to end with `endl` LogMessages